content/en/docs/installation/supported-releases.md

```diff
@@ -17,24 +17,27 @@ release every two months.
 
 ## Supported releases {#supported-releases}
 
-| Release | Release Date |        EOL        | [Supported Kubernetes versions][s] |
-| ------- | :----------: | :---------------: | :--------------------------------: |
-| [1.3][] | Apr 08, 2021 |   [~][]Aug 2021   | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21 |
-| [1.2][] | Feb 10, 2021 | [~][]Jun 11, 2021 | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21 |
+| Release | Release Date |     EOL      | [Supported Kubernetes versions][s] |
+|---------|:------------:|:------------:|:----------------------------------:|
+| [1.4][] | Jun 15, 2021 | Oct 13, 2021 | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21 |
+| [1.3][] | Apr 08, 2021 | Aug 11, 2021 | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21 |
 
 ## Upcoming releases
 
-| Release |   Release Date    |      EOL      | [Supported Kubernetes versions][s] |
-| ------- | :---------------: | :-----------: | :--------------------------------: |
-| [1.4][] | [~][]Jun 11, 2021 | [~][]Oct 2021 | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21 |
+| Release | Release Date |     EOL      | [Supported Kubernetes versions][s] |
+|---------|:------------:|:------------:|:----------------------------------:|
+| [1.5][] | Aug 11, 2021 | Dec 15, 2021 | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21 |
```

Comment:
question: we explicitly want to support 1.22 for 1.5, right? is it worth noting that here?

Reply:
1.22 is planned to be released [on Aug 2021](https://www.kubernetes.dev/resources/release/), would it make sense to only add it then?

Reply:
it would make sense, but I guess I'm thinking "why wait?" - we already know we're gonna support it 100%, so why not add it now? but equally it's not required in this PR

Reply:
Yeah, let's just add it now 😅